buildroot-external/board/raspberrypi/kernel-32b.config

```diff
@@ -0,0 +1,4 @@
+# Config options specific only to 32bit (armv7) builds
```

Comment:
If this Arm spec, can we add this to the file name?

Reply:
It's not ARM-specific, it's rather non-aarch64 (ie 64b) specific :) Not sure if we're going to add more options that would be really 32-bit or ARMv7 specific, but I'll rename it.